### PR TITLE
Add tests for ColorPalette component.

### PR DESCRIPTION
**Pull request by BE-Webdesign**
Add tests for ColorPalette component.

### CODE REVIEW

File: blocks/color-palette/test/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * Internal dependencies
+ */
+import { ColorPalette } from '../';
+
+describe( 'ColorPalette', () => {
+	const colors = [ 'red', 'white', 'blue' ];
+	const currentColor = 'red';
+	const onChange = jest.fn();
+
+	const wrapper = shallow( <ColorPalette colors={ colors } value={ currentColor } onChange={ onChange } /> );
```
Comment (youknowriad) [inline blocks/color-palette/test/index.js]: I can see why you're doing all these initializations outside the `test` calls but an issue with this way of writing the unit tests is that a `test` could alter these global variables and break the following tests. I'd suggest trying to keep each test as isolated as possible.

One way to do it to reinstantiate these variables in `beforeEach`

Comment (BE-Webdesign) [reply blocks/color-palette/test/index.js]: To clarify, change these to let and then rebind them in `beforeEach()` ?

Comment (youknowriad) [reply blocks/color-palette/test/index.js]: yep, I also don't mind repeating these small instantiations from test to test.

Comment (BE-Webdesign) [reply blocks/color-palette/test/index.js]: > yep, I also don't mind repeating these small instantiations from test to test.

I am fine with that as well, but most of the tests in this project don't do that, and everyone else seems to be more in favor of this approach of declaring them at the top and reusing a single declaration. Should we open an issue to figure out what we want to do to be consistent?

Comment (youknowriad) [reply blocks/color-palette/test/index.js]: Not worth an issue IMO but feel free to do so if you think so. :) Just pick the option you like

Comment (BE-Webdesign) [reply blocks/color-palette/test/index.js]: > Just pick the option you like

I like consistency 😄

**Comment by BE-Webdesign** [comment]
Going to leave as is @youknowriad, as this is more in line with how all of the other tests are doing things. We definitely share the same thoughts regarding this, but to have better consistency, I think it might be best to leave this as is?

Let me know if that sounds good to you.

**Review by youknowriad — APPROVED** [review]
Thanks for all these tests :)

**Comment by BE-Webdesign** [comment]
> Thanks for all these tests :)

Thank you for all of your work, Gutenberg is going to be great!

**Pull request merged**